src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/OnTypeFormattingTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Host;
+using Xunit;
+
+namespace Microsoft.VisualStudio.Razor.IntegrationTests
+{
+    public class OnTypeFormattingTests : AbstractRazorEditorTest
+    {
+        [IdeFact]
+        public async Task TypeScript_Semicolon()
+        {
+            var version = await TestServices.Shell.GetVersionAsync(HangMitigatingCancellationToken);
+            if (version < new System.Version(17, 2, 32302, 118))
```

Comment:
This is the version of the validation build that @zkat provided, so surely not the final correct version number, but let the record show, this Works On My Machine™

<img width="192" alt="image" src="https://user-images.githubusercontent.com/754264/156964818-8d7f6c3e-5de1-46cd-9c9e-d43639a8868a.png">